__tests__/cogs.test.js

```diff
@@ -0,0 +1,78 @@
+import pool from '../lib/utils/pool.js';
+import setup from '../data/setup.js';
+import request from 'supertest';
+import app from '../lib/app.js';
+import Cog from '../lib/models/Cog.js';
+
+describe('cog routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('get cogs', async () => {
```

Comment:
Test description should be updated to reflect `creates a new cog` or similar; I almost always read the tests first when I'm grading labs, so documentation-wise it's super helpful for me when I can quickly see what the intention of the test is.